orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -199,6 +195,21 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
     }
   }
 
+  private List<Map> fetchServerGroups(String account, String cloudProvider, Map<String, List<String>> serverGroupsByRegion) {
+    Names names = Names.parseName(serverGroupsByRegion.values().flatten()[0])
+
+    if (serverGroupsByRegion.values().flatten().size() > 1) {
+      def response = oortService.getCluster(names.app, account, names.cluster, cloudProvider)
+      def cluster = objectMapper.readValue(response.body.in().text, Map)
+      return cluster.serverGroups ?: []
+    } else {
+      def region = serverGroupsByRegion.keySet()[0]
+      def response = oortService.getServerGroup(names.app, account, region, names.group)
+      def serverGroup = objectMapper.readValue(response.body.in().text, Map)
+      return [serverGroup]
```

Comment:
Payload should be identical regardless of whether the server group was loaded directly or fetched via a cluster.